tests/baselines/reference/truthinessCallExpressionCoercion2.types

```diff
@@ -129,7 +129,7 @@ function test(required1: () => boolean, required2: () => boolean, b: boolean, op
 >f.apply : (this: Function, thisArg: any, argArray?: any) => any
 >f : () => void
 >apply : (this: Function, thisArg: any, argArray?: any) => any
->parent : Window
+>parent : Window | null

```

Comment:
@orta should we really expect `null` there?
https://developer.mozilla.org/en-US/docs/Web/API/Window/parent#specifications
https://html.spec.whatwg.org/multipage/browsers.html#dom-parent-dev

Reply:
\* for visibility

It was actually legit comment :) -> https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1089
thank you @orta @saschanaz